### PR TITLE
Give current commands exclusive control over all lists

### PR DESCRIPTION
**Pull request by jaivigneshvenugopal**
1) All current commands work in all lists.
2) Included a message to indicate current displayed list.

### CODE REVIEW

File: src/test/java/seedu/address/model/AddressBookTest.java [hunk 1]
```diff
@@ -104,49 +103,21 @@ public void getTagList_modifyList_throwsUnsupportedOperationException() {
...
         public ObservableList<ReadOnlyPerson> getBlacklistedPersonList() {
-            return getBlacklistedPersons(persons).asObservableList();
+            return persons.stream().filter(person -> person.isBlacklisted())
+                    .collect(Collectors.toCollection(FXCollections::observableArrayList));
         }
 
         @Override
         public ObservableList<ReadOnlyPerson> getWhitelistedPersonList() {
-            return getWhitelistedPersons(persons).asObservableList();
+            return persons.stream().filter(person -> person.isBlacklisted())
```
Comment (lawwman) [inline src/test/java/seedu/address/model/AddressBookTest.java]: this is wrong.
change to "person.isWhiteListed()"

File: src/main/java/seedu/address/model/AddressBook.java [hunk 2]
```diff
@@ -209,6 +161,8 @@ public void updatePerson(ReadOnlyPerson target, ReadOnlyPerson editedReadOnlyPer
         persons.setPerson(target, editedPerson);
     }
 
+
+
```
Comment (khooroko) [inline src/main/java/seedu/address/model/AddressBook.java]: why did you add two more blank lines?

Comment (jaivigneshvenugopal) [reply src/main/java/seedu/address/model/AddressBook.java]: why indeed

File: src/test/java/seedu/address/model/AddressBookTest.java (hunk 1 above)
Comment (jaivigneshvenugopal) [reply src/test/java/seedu/address/model/AddressBookTest.java]: Noted. Changed.

File: src/main/java/seedu/address/model/AddressBook.java (hunk 2 above)
Comment (jaivigneshvenugopal) [reply src/main/java/seedu/address/model/AddressBook.java]: Noted. Deleted those lines.

**Review by khooroko — APPROVED** [review]
seems good to me

**Review by lawwman — APPROVED** [review]
Approved.

File: src/main/java/seedu/address/logic/ListObserver.java [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+            return model.getFilteredWhitelistedPersonList();
+
+        default:
+            return model.getFilteredPersonList();
+        }
+    }
+
+    /**
+     * Monitors current displayed list on person list panel.
+     * Updates the current displayed list with {@param predicate}
```
Comment (jelneo) [inline src/main/java/seedu/address/logic/ListObserver.java:50]: i don't think this is how you use `@param` though. But then again I'm unsure so up to you if you want to leave it as it is. small issue.

**Review by jelneo — APPROVED** [review]

File: src/main/java/seedu/address/logic/ListObserver.java (hunk 3 above)
Comment (jaivigneshvenugopal) [reply src/main/java/seedu/address/logic/ListObserver.java:50]: Hmm noted. Will update it correctly in the subsequent PRs. Thanks Jel!
